pkg/combustion/templates/26-embedded-registry.sh.tpl

```diff
@@ -14,9 +39,10 @@ After=network.target
 Type=simple
 User=root
 WorkingDirectory=/opt/hauler
-ExecStartPre=/bin/bash -c "for file in /opt/hauler/*-{{ .RegistryTarSuffix }}; do [ -f \"\$file\" ] && /opt/hauler/hauler store load -f \"\$file\" --tempdir /opt/hauler; done"
-ExecStart=/opt/hauler/hauler store serve registry -p {{ .RegistryPort }}
+ExecStart=/opt/hauler/start-registry.sh
+TimeoutStartSec=300
```

Comment:
What benefit do we see from setting this? Don't we risk never starting in cases where there are either too many container images or too little compute resources?